backend/src/main/kotlin/api/GetStoresHandler.kt

```diff
@@ -0,0 +1,29 @@
+package api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import database.Repository
+import io.javalin.http.Context
+import io.javalin.http.Handler
+
+class GetStoresHandler(private val repo: Repository) : Handler {
+
+    private val objectMapper = ObjectMapper()
+
+    init {
+        objectMapper.registerModule(JavaTimeModule());
+        objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    }
+
+    override fun handle(ctx: Context) {
```

Comment:
I've noticed that you decided to not have pagination on this endpoint and do it on the frontend. are there any advantages of this approach over having the pagination on the backend?

Reply:
The data sources were not reliable, and I did not want to tie or compromise UX based on this unreliability. If we have all the data in frontend, all the data visualization is isolated and not depending on anyone, thus improving response times and user experience. 

This has its disadvantages, as the user needs to fetch a big file (~4MB) at page load, but I'd rather iterate on this issue (maybe splitting it into smaller chunks?).

This way you can also take the most advantage of the library that I've chosen (`material-table`) as you have all the data there already, some possible features such as clustering or giving the user the ability to chose the page size were now possible